tests/data/dataconverter/readers/ellips/eln_data.yaml

```diff
@@ -58,9 +58,6 @@ colnames:
 - Delta
 - err.Psi
 - err.Delta
-definition: NXellipsometry
-definition/@url: https://github.com/FAIRmat-NFDI/nexus_definitions/blob/fairmat/contributed_definitions/NXellipsometry.nxdl.xml
-definition/@version: 0.0.2
```

Comment:
comes with the change above, that these data are autofilled and not manually configured

Reply:
Yes, these are autofilled now.